packages/validation/src/loadbalancers.schema.ts

```diff
@@ -1,12 +1,14 @@
 import { number, object, string, array } from 'yup';
 
+const LABEL_REQUIRED = 'Label is required';
```

Comment:
```suggestion
const LABEL_REQUIRED = 'Label is required.';
```
Looks like omitting the period causes a failure in `load-balancer-routes.spec.ts`. We're pretty inconsistent, but it looks like most of our schema validation errors include the punctuation.